paperqa/docs.py

```diff
@@ -650,7 +650,7 @@ async def aget_evidence(
         for _, llm_result in results:
             session.add_tokens(llm_result)
 
-        session.contexts += [r for r, _ in results if r is not None]
+        session.contexts += [r for r, _ in results]
```

Comment:
This is being discussed in #687. However, the typing change didn't fix the mypy error saying that `r is not None` is a always True statement

Reply:
Cannot do this I think. You can get a `None` back I believe